index.js

```diff
@@ -213,11 +215,18 @@ exports.interpolateName = function interpolateName(loaderContext, name, options)
 		if (typeof context !== 'undefined') {
 			directory = path.relative(context, resourcePath + "_").replace(/\\/g, "/").replace(/\.\.(\/)?/g, "_$1");
 			directory = directory.substr(0, directory.length-1);
+			folders = directory.split('/');
+			ctxFolder = folders[folders.length - 2];
```

Comment:
This is not windows-compatible. Try to use a combination of path.dirname and path.basename instead.